### ISSUE TITLE
I can't access OJS after rebooting the vagrant machine

### ISSUE THREAD

**Issue by dieggpereira** [pkp/vagrant#12]
Well, I apologize if my question is something very simple, but I didn't find any adequate answer to it in forums.

I followed the installation guidelines and everything went well, I was able to access OJS through `localhost:8000` without any errors.

However, after shutting down the vagrant machine (`vagrant halt`) and starting it again (`vagrant up`) I could no longer access `localhost:8000` nor any other port indicated by the terminal starting output.

Not knowing what to do, I reinstalled, tried again and again this happened. I currently installed it for the third time and for use I am using `vagrant suspend` & `vagrant resume`. Is this the only possible way? I use another vagrant machine for AtoM and it works normally after reboot.

Thank you very much in advance.

![Capture d’écran du 2023-09-01 12-58-32](https://github.com/pkp/vagrant/assets/80273980/f4de43e1-1a7a-4266-afdb-803aa7373adc)

**Comment by asmecher** [comment]
@dieggpereira, I suspect this is more of a Vagrant question/problem than an OJS one; somewhere in there will probably be a log entry that indicates the problem. But are you using Vagrant just to explore or something more? Lately we're seeing a lot more emphasis on Docker, and I'd suggest trying that if it's an option.

**Comment by dieggpereira** [comment]
@asmecher , thank you very much! I think I figured out how to work around the problem. I re-read the entire installation script and found `/lib/pkp/tools/travis/prepare-webserver.sh` which was triggered before the tests started. I tried to manually run it after starting the vagrant machine and with that the OJS works normally. I'll add that to the machine start script later, but for now that's enough to solve the problem.
